src-v2/Judge/NTS.Judge.Blazor/Pages/Setup/Contestants/ContestantFormModel.cs

```diff
@@ -11,15 +11,24 @@ public class ContestantFormModel
     public ContestantFormModel()
     {
         // mock data for testing
-        StartTime = TimeSpan.Zero;
+        StartTimeOverride = null;
     }
     public ContestantFormModel(Contestant contestant)
     {
         Id = contestant.Id;
-        TimeSpan? startTime = contestant.StartTimeOverride.DateTime.TimeOfDay;
-        StartTime = startTime;
+        if (contestant.StartTimeOverride != null)
+        {
+            TimeSpan? startTime = contestant.StartTimeOverride.Value.DateTime.TimeOfDay;
```

Comment:
This isn't correct. I'm not sure exactly what time this will return, but probably it will  simply be the time of day for the time-zone it was created in. So if it was in 6:00 in Sofia time then it could simply return 6:00 when you open it up in UK. You need to convert the DateTimeOffset value to the current timezone. This isn't a big issue right now, but eventually - when we have web APIs and databases it could become one.

Reply:
Also implicit vars. Explicit declaration is only acceptable if it is necessary.